scripts/error-codes/codes.json

```diff
@@ -143,5 +143,46 @@
   "141": "Expected onSetChildren() to fire for a container child before its parent includes it in onSetChildren().",
   "142": "Expected onBeforeMountComponent() parent and onSetChildren() to be consistent (%s has parents %s and %s).",
   "143": "React.Children.only expected to receive a single React element child.",
-  "144": "React.PropTypes type checking code is stripped in production."
-}
\ No newline at end of file
+  "144": "React.PropTypes type checking code is stripped in production.",
```

Comment:
I almost removed it when I rebased for @davidblurton, but it wasn't clear if this should be excluded. There is a test that relies on testing the new error (and error code) in `ReactDOMProduction`, won't that test potentially fail if `codes.json` isn't updated now since the code will potentially change?